primefaces/src/main/resources/META-INF/resources/primefaces/core/core.js

```diff
@@ -421,19 +421,29 @@
                         return;
                     }
 
-                    button.removeClass('ui-state-loading');
-
-                    if (widget.hasOwnProperty('enable')
-                        && widget.cfg.disableOnAjax !== false
-                        && !widget.cfg.disabledAttr) {
-                        widget.enable();
-                    }
-
-                    button.find('.ui-icon-loading').remove();
+                    setTimeout(PrimeFaces.buttonEndAjaxDisabled,
```

Comment:
```suggestion
                    PrimeFaces.queueTask(PrimeFaces.buttonEndAjaxDisabled,
```

Reply:
How can you pass parameters using this? I'm now getting `Uncaught TypeError: Cannot read properties of undefined (reading 'removeClass')`

Reply:
Alright, I think I've got it... @melloware please have a look